hs_app_timeseries/tests/test_timeseries_metadata.py

```diff
@@ -66,133 +73,502 @@ def tearDown(self):
         Method.objects.all().delete()
         ProcessingLevel.objects.all().delete()
         TimeSeriesResult.objects.all().delete()
```

Comment:
If you inherit from django's own `TestCase` it will clean these up. if you get transaction problems, try the Django `TransactionTestCase` instead. These deletes should go, either way.